articles/building-documentation-websites-with-docusaurus-v2/index.md

```diff
@@ -0,0 +1,201 @@
+
+Run the following command.
+
+```bash
+npx @docusaurus/init@latest init my-blog classic
+```
+
+### File structure
+
+The command generates a directory known as my-blog. Navigate into the directory to see the generated files. Some of the files and folders that you should pay attention to are `docusaurus.config.js`, the `src` folder, and the `blog` folder. Let us run and build the project and see how it looks like without any customization.
+
+```bash
+npm run start
+```
+
+```bash
+npm run build
+```
+
+Visit [http://localhost:3000/](http://localhost:3000/) on your browser to view how uncustomized websites look like.
+
+![Uncustomised Website](/engineering-education/building-documentation-websites-with-docusaurus-v2/image1.png)
+
+### Modifying the contents of the homepage
+
+To change the contents of the homepage we are going to edit the `docusaurus.config.js` file.
+
+```javascript
+module.exports = {
+  title: '', // Here goes the title of your website
+  tagline: '', // Here goes the tagline of your blog
+  url: '', // Here goes the link to your site
+  baseUrl: '/', // baseurl of the site
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: '',  // link to the favicon
+  organizationName: '', // Usually your GitHub org/user name.
+  projectName: '', // Usually your repo name.
+
+  // This block of command is for customising your naviagtion bar
+  themeConfig: {
+    navbar: {
+      title: 'My Site',
+      logo: {
+        alt: 'My Site Logo',
+        src: 'img/logo.svg',
+      },
+      items: [
+        {
+          type: 'doc',
+          docId: 'intro',
+          position: 'left',
+          label: 'Tutorial',
+        },
+        {to: '/blog', label: 'Blog', position: 'left'},
+        {
+          href: '', 
+          label: '',
+          position: 'right',
+        },
+      ],
+    },
+
+    // This block of code is for customizing the footer
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'Community/socials',
+          items: [
+            {
+              label: 'Github',
+              href: '', // Github link
+            },
+            {
+              label: 'Instagram',
+              href: '', // Instagram link
+            },
+            {
+              label: 'Twitter',
+              href: '', // Twitter link
+            },
+          ],
+        },
+        
+      ],
+      copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+    },
+  },
+};
+```
+
+### Modifying the style of the homepage
+
+To change the look and feel of the Homepage navigate to the `src` folder, then the `CSS` folder, and open the `custom.css` file. If you want to style your website further this is the file to add the CSS.
+
+```css
```

Comment:
Please include custom css and explain how the changes reflect on the page.
the below CSS is already pre-populated in the `custom.css` file